frontend/src/views/WheresMyGeneV2/components/HeatMap/index.tsx

```diff
@@ -356,6 +356,39 @@ export default memo(function HeatMap({
     selectedCellTypes: filteredCellTypes,
   });
 
+  /**
+   * All tissue cell types to render in YAxisCharts
+   */
+  const allTissueCellTypes = useMemo(() => {
```

Comment:
Keep the code DRY and extract calculation to its useMemo, so we don't need to use `memoize`